index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare function humanInterval(time?: string): number | undefined;
+
+declare namespace humanInterval {
```

Comment:
not quite sure what this namespace definition is for? is the languagemap something this library exports? and should the types of it be overwriteable?

the only export this library has according to documentation is the function "humanInterval(..)" as default export..

Reply:
Right, that used to be a thing but I forgot it got already refactored out, here:

https://github.com/agenda/human-interval/pull/37/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346L66

Reply:
Removed in https://github.com/agenda/human-interval/pull/43/commits/ed1c53691274682e96acae2b4b9434106b37e1c2 — Thanks for pointing it out!